scenarios/kubernetes_e2e.py

```diff
@@ -278,7 +278,10 @@ def main(args):
 
     cluster = cluster_name(args.cluster, args.tear_down_previous)
     runner_args.append('--cluster=%s' % cluster)
-    runner_args.append('--gcp-network=%s' % cluster)
+    network = args.network
+    if not network:
+        network = cluster
+    runner_args.append('--gcp-network=%s' % network)
```

Comment:
```suggestion
    if args.network:
       runner_args.append('--gcp-network=%s' % network)
    else:
          runner_args.append('--gcp-network=%s' % cluster)
```

Reply:
WDYT?

Reply:
please stop using this script versus using kubetest2